.github/workflows/ci.yml

```diff
@@ -156,6 +159,14 @@ jobs:
         run: cd build && make shellcheck
       - id: flake8_check
         run: cd build && make flake8
+      - id: coverage
```

Comment:
I thought it would be useful for the developers to see the test coverage in case of a newly implemented feature, but I suppose that could be done manually on their local environment and the CI would only generate on every push on main. I updated the PR with this condition in ba58ef2cd8438a314eb6bfd5ab25e46335e87be7

Reply:
The condition seems no different from all of our typical runs, so it may be better to just omit it. That is if the build time overhead is low enough.

Reply:
My bad, I misread the condition to be OR instead of AND. That's fine then, sorry for the confusion.